src/Authentication/Authentication/Cmdlets/ConnectMgGraph.cs

```diff
@@ -100,6 +100,9 @@ public class ConnectMgGraph : PSCmdlet, IModuleAssemblyInitializer, IModuleAssem
         [Parameter(ParameterSetName = Constants.EnvironmentVariableParameterSet, Mandatory = false, HelpMessage = HelpMessages.EnvironmentVariable)]
         public SwitchParameter EnvironmentVariable { get; set; }
 
+        [Parameter(ParameterSetName = Constants.UserParameterSet, Mandatory = false, HelpMessage = HelpMessages.SigninUi)]
+        public SigninUi SigninUi { get; set; }
```

Comment:
We should consider using a global variable like `$EnableWAMForMSGraph` or something better that customers can use to opt-in/out of WAM. This is needed to address the feedback we got on using `-SigninUi` parameter. Having a global way of opting-in/out of WAM will eliminate the need for customers to specify WAM as their auth broker for every auth call via `Connect-MgGraph -SigninUI`.

We should default to WAM for auth on Windows once `Azure.Identity.BrokeredAuthentication` goes GA.

Reply:
Thank you for this feedback. The suggestion has been implemented.